viseron/webserver/assets/templates/about.html

```diff
@@ -0,0 +1,24 @@
+{% extends "layout.html" %}
+{% block title %}About{% end %}
+{% block content %}
+<h1 class="title">Viseron - Self-hosted NVR with object detection</h1>
+<h2 class="subtitle">Version: {{ version }}</h2>
```

Comment:
Its fine to leave this as it is now, i need to figure something out with the versioning.